### PR TITLE
Update Config and readmes

### PR DESCRIPTION
**Pull request by joshua-pe**
Updated config URL to link the new docs, changed the readme to .md since homeassistant doesnt seem to support rst. I have also added your name as a mainainer (just shows who made it)

### CODE REVIEW

File: repository.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "Photons",
-  "url": "https://github.com/delfick/photons-homeassistant"
+  "url": "https://github.com/delfick/photons-homeassistant",
+  "maintainer": "Delfick"
```
Comment (delfick) [inline repository.json]:
```suggestion
  "maintainer": "delfick"
```

File: photons-interactor/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Photons interactor
```
Comment (delfick) [inline photons-interactor/README.md]:
```suggestion
# Photons interactor
```

and remove the underlines

**Review by delfick — COMMENTED** [review]
silly homeassistant, rst is far superior !!! :p

**Comment by Final-Hawk** [comment]
Also, forgot to add here so can you add a CHANGELOG.md and just put https://photons.delfick.com/apps/interactor/changelog.html in it?

File: repository.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "Photons",
-  "url": "https://github.com/delfick/photons-homeassistant"
+  "url": "https://github.com/delfick/photons-homeassistant",
+  "maintainer": "delfick"
```
Comment (delfick) [inline repository.json]: Actually, better would be

```suggestion
  "maintainer": "Stephen Moore <github@delfick.com>"
```

**Comment by delfick** [comment]
yeap, that's the changelog

**Comment by joshua-pe** [comment]
> yeap, that's the changelog

Yes, but create a file in [here](https://github.com/delfick/photons-homeassistant/tree/main/photons-interactor) called "CHANGELOG.md" and just put the link to the read the docs changelog.

[Heres ](https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog) the homeassistant docs on it, however some people just put the link to their website that has the changelog, which is fine and easier for us.

EDIT: managed to create commits, was having trouble before

File: repository.json (hunk 3 above)
Comment (joshua-pe) [reply repository.json]: Yup, thats better

**Comment by delfick** [comment]
sure

File: photons-interactor/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Photons interactor
+# Photons interactor
 ==================
```
Comment (delfick) [inline photons-interactor/README.md]:
```suggestion

```

**Comment by delfick** [comment]
looks good. Can I get you to also change it to 0.8.6 to use the version I just released and then I'll merge

**Comment by joshua-pe** [comment]
Can do. Done. Thanks

**Comment by delfick** [comment]
thanks :)

**Pull request merged**